### PR TITLE
EZ-350: Article Autosave

### PR DESCRIPTION
**Pull request by saurabh-2k17**

### CODE REVIEW

File: config/install/autosave_form.settings.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-interval: 600
+interval: 1000
```
Comment (vaibhavjain-in) [inline config/install/autosave_form.settings.yml]: This needs to be 10000
10000 ms = 10 s

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@saurabh-2k17 we need to change this.

File: config/install/autosave_form.settings.yml (hunk 1 above)
Comment (saurabh-2k17) [reply config/install/autosave_form.settings.yml]: Sorry for the typo.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**